### PR TITLE
Stabilization

### PR DESCRIPTION
**Pull request by VideoGameSmash12**
This will hopefully stabilize the plugin and fix a lot of bugs in the process.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_whitelist.java [hunk 1]
```diff
@@ -198,10 +197,21 @@ else if (args[0].equals("remove"))
...
-            names.add(String.valueOf(name));
+            names.add(player.getName());
         }
         return names;
     }
+
+    public int purge()
+    {
+        int removed = 0;
+        for (OfflinePlayer player : server.getWhitelistedPlayers())
```
Comment (Telesphoreo) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_whitelist.java:210]: Does this account for people who are online as well?

Comment (VideoGameSmash12) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_whitelist.java:210]: Yes.

Comment (Telesphoreo) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_whitelist.java:210]: Then everything else looks fine to me.

**Review by Telesphoreo — APPROVED** [review]
Everything looks good, nice work

**Pull request merged**